File: Data Structures & Algorithms/online-stock-span/submission-1.py
class StockSpanner:

    def __init__(self):
        self.index = 0
        self.stack = []

    def next(self, price: int) -> int:
        self.index += 1
        while self.stack and self.stack[-1][0] <= price:
            self.stack.pop()
        
        prevIndex = 0 if not self.stack else self.stack[-1][1]
        self.stack.append([price, self.index])
        return self.index - prevIndex


# Your StockSpanner object will be instantiated and called as such:
# obj = StockSpanner()
# param_1 = obj.next(price)

File: Data Structures & Algorithms/design-hashset/submission-1.cpp
class MyHashSet {
private:
    vector<bool> numbers = vector<bool>(1000001, false);
public:
    MyHashSet(){
    }
    
    void add(int key) {
        this->numbers[key] = true;
    }
    
    void remove(int key) {
        this->numbers[key] = false;
    }
    
    bool contains(int key) {
        return this->numbers[key];
    }
};

/**
 * Your MyHashSet object will be instantiated and called as such:
 * MyHashSet* obj = new MyHashSet();
 * obj->add(key);
 * obj->remove(key);
 * bool param_3 = obj->contains(key);
 */